activestorage/lib/active_storage/attached/many.rb

```diff
@@ -70,6 +70,17 @@ def attached?
       attachments.any?
     end
 
+    # Returns the combined size in bytes of all attached blobs.
+    #
+    #   class Gallery < ApplicationRecord
+    #     has_many_attached :photos
+    #   end
+    #
+    #   Gallery.new.photos.bytes_size # => 2048
+    def bytes_size
+      blobs.pluck(:byte_size).sum
```

Comment:
Can you not do this?

```
blobs.sum(:byte_size)
```

That'd return 1 row 1 column from the database instead of N rows 1 column, because the summing would occur in the database.